lib/rules/best-practises/no-empty-blocks.js

```diff
@@ -16,32 +20,53 @@ const meta = {
           description: 'empty block in constructor without parent initialization',
           code: 'constructor () {}',
         },
+        {
+          description: 'empty modifier with only _ (when allowEmptyModifiers is false)',
+          code: 'modifier onlyOwner { _; }',
+        },
       ],
       good: [
         { description: 'empty receive function', code: 'receive () external {}' },
         { description: 'empty fallback function', code: 'fallback () external {}' },
         {
-          description: 'empty constructor with member initialization list',
-          code: 'constructor(uint param) Foo(param) Bar(param*2) {}',
+          description: 'empty constructor with parent init',
+          code: 'constructor(uint x) Base(x) {}',
         },
+        { description: 'empty modifier with _ allowed', code: 'modifier onlyOwner { _; }' },
+        { description: 'empty try-catch block allowed', code: 'try foo() {} catch {}' },
       ],
     },
     notes: [
       {
-        note: 'The rule ignores an empty constructor by default as long as parent contracts are being initialized. See "Empty Constructor" example.',
+        note: 'Empty constructor is ignored if the constructor has parent-initialization modifiers.',
+      },
+      {
+        note: 'Use allowEmptyModifiers / allowEmptyCatch to skip warnings for underscore-only modifiers / empty catch blocks.',
       },
     ],
   },
 
   recommended: true,
-  defaultSetup: 'warn',
-
-  schema: null,
+  defaultSetup: ['warn', { allowEmptyModifiers: false, allowEmptyCatch: false }],
+  schema: {
+    type: 'object',
+    properties: {
+      allowEmptyModifiers: { type: 'boolean' },
+      allowEmptyCatch: { type: 'boolean' },
+    },
+  },
 }
 
 class NoEmptyBlocksChecker extends BaseChecker {
   constructor(reporter) {
     super(reporter, ruleId, meta)
+
+    const [, options = {}] = Array.isArray(this.reporter?.config?.[ruleId])
```

Comment:
what was the reason for doing this instead of receiving the config as a parameter? (which would mean editing the index file for best-practises, but it's what we are currently doing for all rules).